.travis_script.sh

```diff
@@ -17,6 +17,7 @@ then
     exit $conf
 fi
 export PYTHONPATH=$(pwd)/tools/pylib/:$PYTHONPATH
+export MAKEFLAGS="-j 2"
 time make || exit
 time make check-unit-tests || exit
```

Comment:
I think the integrated tests target in the makefile doesn't list each job independently -- it just runs the `test_suite_make` and `test_suite` scripts with no dependencies, so I don't think it will actually end up with multiple jobs running at the same time. The `-j 2` on all the `make` test commands is essentially pointless -- it only impacts the main compilation I think.